docs/components/concepts/resource-deletion.md

```diff
@@ -0,0 +1,74 @@
+---
+id: resource-deletion
+title: "Resource deletion"
+description: "Use resource deletion to remove process definitions and decision requirements graphs from Camunda."
+---
+
+You can use resource deletion to remove resources from the Camunda Platform. There are two types of resources that can be deleted:
+
+1. Process definitions
+2. Decision Requirements Graphs (DRG)
+
+You can delete a resource using [Operate](../../components/operate/operate-introduction.md), or by sending
```

Comment:
Would briefly mention *how* you can delete a resource using Operate.

Reply:
@mihail-ca Do you have an issue to document how to use the feature using Operate? If so, could you make sure the link on this page is updated accordingly?

Reply:
Hello, this is our Operate epic for this topic: https://github.com/camunda/operate/issues/3972
The relevant issues for documentation are on the front-end side, since the back-end changes are only available via Operate UI. I think this issue explains how to do it: https://github.com/camunda/operate/issues/4019

Reply:
The above is for deleting decision definition - BTW this can be a point of confusion. In Operate one cannot select decision requirements, only a decision definition.

For Process Definition Deletion, the relevant epic and issue on Operate are:
https://github.com/camunda/operate/issues/3971
https://github.com/camunda/operate/issues/4126

Note though that Process Definition Deletion is still work in progress.

Reply:
@mihail-ca Will there be a page similar to this one? https://docs.camunda.io/docs/components/operate/userguide/process-instance-modification/

Where can I find the issue for this?

Reply:
Hello, I am not sure about this. Maybe @pedesen knows, he is DRI for resource deletion on Operate side.

Reply:
@remcowesterhoud @mihail-ca I will create an issue for this in the camunda-platform-docs repo

Reply:
https://github.com/camunda/camunda-platform-docs/issues/2507